### ISSUE TITLE
Multiple popover's on the same element

### ISSUE THREAD

**Issue by MickL** [ng-bootstrap/ng-bootstrap#2266]
Is it possible to have multiple, different popover's on the same element? E.g. i would like to have one for hover and a different for click.

**Comment by pkozlowski-opensource** [comment]
Nope, not currently. It is an interesting use-case, though, so marking it as a feature request. Still, I don't think we will come to it anytime soon so your best bet for now would be to build a custom solution on top of the primitives we are using in the popover (`PopupService `, `positionElements `).

The code of the popover is not _that_ complex when using mentioned services: https://github.com/ng-bootstrap/ng-bootstrap/blob/master/src/popover/popover.ts

**Comment by MickL** [comment]
Nested div's would be a workaround, right? Or simply changing the content of the popover on hover/click which.

**Comment by ppazos** [comment]
I just reached the need of this in BS4, because I have a button that on hover displays some info and on click displays a popover with a form to perform an action. Right now only the info popover works.
